RAT/controls.py

```diff
@@ -0,0 +1,1322 @@
+import tabulate
+from typing import Union, Any, Literal
+from RAT.utils.enums import ParallelOptions, Procedures, DisplayOptions, BoundHandlingOptions, StrategyOptions
+
+
+class BaseProcedure:
+    """
+    Defines the base class with properties used in 
+    all five procedures.
+    """
+
+    def __init__(self,
+                 parallel: Literal['single', 'points', 'contrasts', 'all'] = ParallelOptions.Single,
```

Comment:
In the initial commit, I was using `parallel: Union[str, ParallelOptions] = ParallelOptions.Single` as the type hint . The reason for that choice was the error raised in MATLAB specifies that you can use either enum or string when setting parallel value. I changed it to Literal because I thought that it made it more clearer what values are accepted for that variable. I can change the typing to just `ParallelOptions`.

Reply:
Ok, this is another thing that will be resolved with pydantic.